services/reference/arbitrum/index.md

```diff
@@ -54,6 +54,6 @@ Select one of the following options to get started with the Arbitrum network:
 
 No personal information is sent as part of partner requests, only information necessary to fulfill your API request. This means that Infura's partner service provider can service your request, but not store the content of your request.
 
-For any parameters required in an RPC request, these could be the type that describe the method, addresses, gas, and session.
+For any parameters required in an RPC request, these could be the types that describe the method, addresses, gas, and session.
 
```

Comment:
```suggestion
```

This sentence is a little confusing, I think we can just remove it altogether. Can you also remove it from the Avalanche, Optimism, and Polygon PoS index pages?

Reply:
done👍